corehq/apps/app_manager/views/forms.py

```diff
@@ -391,7 +391,15 @@ def new_form(request, domain, app_id, module_unique_id):
     lang = request.COOKIES.get('lang', app.langs[0])
     form_type = request.POST.get('form_type', 'form')
     case_action = request.POST.get('case_action', 'none')
-    name = _("Register") if case_action == 'open' else (_("Followup") if case_action == 'update' else "Survey")
+
+    if case_action == 'open':
+        name = _('Register')
+    elif case_action == 'update':
+        name = _('Followup')
+    elif form_type == 'lesson':
```

Comment:
What would you think of basing this on the module type (training or not) instead of having a separate form type? I realize "form type" doesn't actually mean much here, but reading through this code I initially thought lessons were genuinely a different kind of form.

Reply:
updated